### PR TITLE
chore: upgrade Fluentd to v1.15.3-sumo-0

### PR DESCRIPTION
**Pull request by astencel-sumo**
Backports #2745.

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -88,9 +88,10 @@ The following table displays the tested Kubernetes and Helm versions.
 
 The following matrix displays the tested package versions for our Helm chart.
 
-| Sumo Logic Helm Chart | kube-prometheus-stack/Prometheus Operator | FluentD | Fluent Bit                          | Falco  | Metrics Server | Telegraf Operator | Tailing Sidecar Operator | OpenTelemetry Operator |
+| Sumo Logic Helm Chart | kube-prometheus-stack/Prometheus Operator | Fluentd | Fluent Bit                          | Falco  | Metrics Server | Telegraf Operator | Tailing Sidecar Operator | OpenTelemetry Operator |
```
Comment (sumo-drosiek) [inline deploy/README.md:91]: `Fluentd` or `FluentD`? :confused:

**Comment by sumo-drosiek** [comment]
Table formatter is complaining :sob:

File: deploy/README.md (hunk 1 above)
Comment (astencel-sumo) [reply deploy/README.md:91]: https://www.fluentd.org/

image

Comment (sumo-drosiek) [reply deploy/README.md:91]: I feel like we are going back and forth :joy_cat:

**Review by sumo-drosiek — APPROVED** [review]

**Pull request merged**